features/testbot/via.feature

```diff
@@ -148,6 +148,24 @@ Feature: Via points
             | a,d,c     | abc,bd,bd,bd,abc,abc |
             | c,d,a     | abc,bd,bd,bd,abc,abc |
 
+    # See issue #2349
```

Comment:
No, its an addition. Your test is still in there, but is only for exiting oneways, not entering ones.